### ISSUE TITLE
LB Photo Realism Reload! incompatible

### ISSUE THREAD

**Issue by thepriz** [FiniteReality/embeddium#430]
### Bug Description

Updates after embeddium-0.3.19+mc1.20.1-all have a terrible studder during play. As you walk around the game studders with LB Photo Realism Reloaded installed.

### Reproduction Steps

Install latest Embeddium and LB Photo Realism and run the game. If you uninstall Embeddium the games works with out the studder.

### Log File

[2024-12-18-3.log](https://github.com/user-attachments/files/18190911/2024-12-18-3.log)

**Comment by FiniteReality** [comment]
Just to confirm, the performance issues also stop if you remove LB Photo Realism Reloaded? Furthermore, the log you sent contains a bunch of other mods (and, in fact, appears to have crashed on startup) - could you also confirm whether the issue reproduces with the minimal number of mods needed to run that combination?

**Comment by thepriz** [comment]
Yes, If I remove LB Photo Realism Reloaded, I don't get the stuttering effect. I even tested it with just Embeddium and Oculus installed, with only the LB Photo Realism Reloaded resource pack installed. After the version listed above, I get the stuttering effect.

It does the same thing in 1.21.1 with just Embeddium 1.0.15+mc1.21.1 and the LB Photo Realism Reloaded Resource Pack installed.In 1.21.1. It works fine with Sodium. Both testbeds are on Neoforge 21.1.186.
